google_health/params.json.template

```diff
@@ -3,7 +3,7 @@
   "export_dir": "./receiving",
   "cache_dir": "./cache",
   "data_dir": "./data",
-  "start_date": "2020-01-05",
+  "start_date": "",
```

Comment:
What should the value of this parameter be in production now?

Reply:
This should be the export start date. So when you run the pipeline, what't the first date of output files

Reply:
I guess I meant to ask whether there should be a default value here.  But it sounds like no.